docs/computing/connecting/ssh-unix.md

```diff
@@ -3,12 +3,36 @@
 --8<-- "auth-update-ssh.md"
 
 On Unix-based systems like macOS and Linux, it is recommended to connect to CSC
-supercomputers using the pre-installed terminal program.
+supercomputers using the pre-installed terminal program. The OpenSSH client
+typically comes pre-installed on macOS and Linux systems.
+
+## Generating SSH keys
+
+On macOS and Linux, you can use the `ssh-keygen` command-line utility for
```

Comment:
Reading the page this section is now not really clear for somebody who does not know what ssh keys are. Why do we need them? Should we add a sentance or two that says that to access supercomputers you need to set up ssh keys. 

Perhaps merge this section and the next on copying to supercomputers?